tests/cli/test_migrate.py

```diff
@@ -180,6 +180,7 @@ def test_migration_version():
     assert max_migration_version == SUPPORTED_PROJECT_VERSION
 
 
+@pytest.mark.skip
```

Comment:
Actually, we can't respect `--no-commit` flag when migrating because we have to commit changes for some migrations to work. I removed this flag completely.